Rule/Symfony2/ControllerMethodName.php

```diff
@@ -27,8 +27,15 @@ public function apply(AbstractNode $node)
             return;
         }
 
+        $allowedMethodNames = explode($this->getStringProperty('delimiter'), $this->getStringProperty('allowedMethodNames'));
+
         /** @var MethodNode $method */
         foreach ($node->getMethods() as $method) {
+            // Check if method is whitelisted
+            if (in_array($method->getImage(), $allowedMethodNames)) {
```

Comment:
Thx for your pull request. Can you please add true === in_array(..... Then I can merge the changes.